Functions/xASL_adm_GetPopulationSessions.m

```diff
@@ -19,6 +20,12 @@
 %   5. Check and provide warning of number of sesssions differs per subject
 
 %% 1. Administration
+if ~isfield(x, 'SESSIONS')
+    SESSIONS = 'ASL_1';
```

Comment:
But what if some subjects don't have a first session?

Reply:
Good point, should the default be an empty `SESSIONS` output then?

Reply:
If you load a dataset that has at least one sessions, you will definitely get a SESSIONS field, even if you did not manually define SESSIONS (`[x] = ExploreASL_Master(my-bids-dataset,0,0);`).

Reply:
@MichaelStritt but that would mean that this condition above never really happens. So we can define an empty session there?!